docs/.eleventy.js

```diff
@@ -457,7 +461,8 @@ module.exports = function(eleventyConfig) {
                 req.url += ".html";
             }
             return next();
-        }
+        },
+        files: "./_site/css/**/*.css"
```

Comment:
This one still doesn't seem to reload on the CSS changes still

Reply:
We tried to do the same thing with playground but it eleventy didn't detect file changes inside `_site` directory. Try with the postcss output in the same place `/assets/css` rather than inside the  `_site`. Eleventy detects the changes inside `css` directory and copies the file changes into `_site`.


Also you seem to use the `build:postcss` command for start script? In that case the postcss command is executed only once and when the scss file changes the output of `postcss` will be over written by that of `scss`?

Reply:
I added the `watch:postcss` script, so the `npm run start`  also watches for the SCSS file changes, builds, then runs PostCSS

Reply:
`"watch:postcss": "postcss src/assets/css -d src/assets/css --watch --poll"`

> Try with the postcss output in the same place /assets/css rather than inside the _site. Eleventy detects the changes inside css directory and copies the file changes into _site.


Even though we mention the `files: "./_site/css/**/*.css"` it doesnt detect 😞